src/NServiceBus.Core/MessageSession.cs

```diff
@@ -26,9 +26,9 @@ public Task Publish(object message, PublishOptions options)
             return messageOperations.Publish(context, message, options);
         }
 
-        public Task Publish<T>(Action<T> messageConstructor, PublishOptions publishOptions)
+        public Task Publish<T>(Action<T> messageConstructor, PublishOptions options)
```

Comment:
Given https://github.com/Particular/NServiceBus/pull/5772 should we roll this back?